allennlp_models/classification/dataset_readers/stanford_sentiment_tree_bank.py

```diff
@@ -153,3 +153,7 @@ def make_token(t: Union[str, Token]):
                     sentiment = "1"
             fields["label"] = LabelField(sentiment)
         return Instance(fields)
+
+    @overrides
+    def apply_token_indexers(self, instance: Instance) -> None:
+        instance.fields["tokens"]._token_indexers = self._token_indexers
```

Comment:
This is equivalent but preferred:

```suggestion
        instance.fields["tokens"].token_indexers = self._token_indexers
```

Reply:
ok